src/airdrop/src/lib.rs

```diff
@@ -486,5 +490,44 @@ fn remove_admins(principals: Vec<Principal>) -> CustomResult<()> {
     })
 }
 
+/// Get the aidrop status - the actual state and whether tokens have been transferred
+#[query(guard = "caller_is_admin")]
+fn get_state_rewards() -> CustomResult<Vec<EthereumTransaction>> {
```

Comment:
you can remove the `state_` in each of these function to simplify the naming

e.g. `get_state_admins` -> `get_admins`